stories/checkbox/checkbox.stories.js

```diff
@@ -92,6 +92,28 @@ On desktop screens, the checkbox appears smaller and uses the <code>fd-checkbox\
     }
 };
 
+export const required = () => `
+<fieldset class="fd-fieldset">
+    <legend class="fd-fieldset__legend">Required checkbox</legend>
+    <div class="fd-form-group">
+        <div class="fd-form-item">
+            <input type="checkbox" class="fd-checkbox fd-checkbox--compact" id="Ai4ez611c">
+            <label class="fd-form-label--required fd-checkbox__label fd-checkbox__label--compact" for="Ai4ez611c">
```

Comment:
Could you change `id`/`for` ? It seems to be duplicated

Reply:
fixed